app/src/main/java/com/framgia/fsalon/screen/customerinfo/CustomerInfoViewModel.java

```diff
@@ -0,0 +1,67 @@
+package com.framgia.fsalon.screen.customerinfo;
+
+import android.support.annotation.StringDef;
+import android.support.v7.app.AppCompatActivity;
+
+import com.framgia.fsalon.R;
+import com.framgia.fsalon.screen.info.InfoUserFragment;
+
+/**
+ * Exposes the data to be used in the CustomerInfo screen.
+ */
+public class CustomerInfoViewModel implements CustomerInfoContract.ViewModel {
+    private static final String INFO = "Info";
+    private static final String BILL = "Bill";
+    private static final String PICTURE = "Picture";
+    private CustomerInfoContract.Presenter mPresenter;
+    private CustomerInfoAdapter mPagerAdapter;
+    private int[] mImageSource = {R.drawable.ic_user, R.drawable.ic_list, R.drawable.ic_picture};
+
+    public CustomerInfoViewModel(AppCompatActivity activity) {
+        mPagerAdapter = new CustomerInfoAdapter(activity.getSupportFragmentManager());
+        mPagerAdapter.addFragment(InfoUserFragment.newInstance(), Tab.INFO);
+        mPagerAdapter.addFragment(InfoUserFragment.newInstance(), Tab.BILL);
+        mPagerAdapter.addFragment(InfoUserFragment.newInstance(), Tab.PICTURE);
+    }
+
+    @Override
+    public void onStart() {
+        mPresenter.onStart();
+    }
+
+    @Override
+    public void onStop() {
+        mPresenter.onStop();
+    }
+
+    @Override
+    public void setPresenter(CustomerInfoContract.Presenter presenter) {
+        mPresenter = presenter;
+    }
+
+    public CustomerInfoAdapter getPagerAdapter() {
+        return mPagerAdapter;
+    }
+
+    public void setPagerAdapter(CustomerInfoAdapter pagerAdapter) {
+        mPagerAdapter = pagerAdapter;
+    }
+
+    /**
+     * Title of pages
+     */
+    @StringDef({INFO, BILL, PICTURE})
+    public @interface Tab {
+        String INFO = "Info";
```

Comment:
ERROR: 'member def type' have incorrect indentation level 8, expected level should be 4.